sgtm/apollo/README.md

```diff
@@ -0,0 +1,196 @@
+# gPS Apollo
```

Comment:
I would add something about the rate limits on the Google Sheet API:

https://developers.google.com/sheets/api/limits

@HalleyInteractive was this the same limit that we discussed? Let me know if anything else needs highlighting.

Reply:
Thank you so much for sharing a good point!!!
I added a sentence as well as its devsite link.

Thank you!